helpers/jwthelper.py

```diff
@@ -0,0 +1,120 @@
+import jwt
+from jwt import ExpiredSignatureError, InvalidTokenError
+from datetime import datetime, timedelta
+from typing import Optional, Set, Dict, Any
+import time
+
+
+class TokenVerificationException(Exception):
+    """Exception raised for token verification errors."""
+
+
+class JwtHelper:
+    "JWT Helper class."
+
+    class VerifyJwtOptions:
+        """JWT Verification Options"""
+
+        DEFAULT_CLOCK_SKEW_MS = 5000
+
+        def __init__(
+                self,
+                key: Any,
+                audience: Optional[str] = None,
+                authorized_parties: Optional[Set[str]] = None,
+                clock_skew_in_ms: int = DEFAULT_CLOCK_SKEW_MS
+        ):
+            if key is None:
+                raise ValueError("key cannot be None.")
+            self.key = key
+            self.audience = audience
+            self.authorized_parties = authorized_parties if authorized_parties else set()
+            self.clock_skew_in_ms = clock_skew_in_ms
+
+    @staticmethod
+    def generate_token(
+            key,
+            subject = None,
+            audience = None,
+            expiration = None,
+            not_before = None,
+            issued_at = None,
+            azp = None
+    ):
+        """Generates a JWT token with the given claims."""
+
+        payload = {
+            k: v for k, v in {
+                'sub': subject,
+                'aud': audience,
+                'exp': expiration,
+                'nbf': not_before,
+                'iat': issued_at,
+                'azp': azp
+            }.items() if v is not None
+        }
+        return jwt.encode(payload, key, algorithm="HS256")
```

Comment:
I don't quite understand why we are adding this utility. I don't think there's any scenario where the user should be generating a JWT, clerk does all the generation with our service

Reply:
I have moved the `generate_token` helper function to the tests and updated the Example usage so you can adapt the payload to your needs